docroot/themes/custom/mass_theme/overrides/css/basic-html.css

```diff
@@ -49,3 +49,8 @@ figure figcaption {
     margin-bottom: 1em;
   }
 }
+
+/* Set FormStack form direction if it has rtl class */
+form.fsForm.rtl {
```

Comment:
this part looks good to me! 
Url to test: https://pr1733-sj9br0xvxnvjdm9lmdthnnacfubyjr4m.tugboatqa.com//forms/tqdym-mlahzat-bkhsws-ljnt-mrajt-alansaf-walmsalt-fy-snadyq-alamwal-alfdralyt